DOCSP-34038: agg tutorials landing (#830)
mongodb/docs-node: source/aggregation-tutorials/one-to-one-join.txt

Introduction
------------

In this tutorial, you can learn how to use the {+driver-short+} to
perform a one-to-one join. A one-to-one join occurs when a document
in one collection has a field value that matches a single document in
another collection with the same field value. The aggregation matches
these documents on the field value and combines information from both
into one document.

.. tip::

   A one-to-one join does not require the documents to have a

>    One-to-one relationship. To learn more about this data relationship,

Reviewer: I don't think capitalization is necessary here - only in reference to the title of the linked doc.
```suggestion
   one-to-one relationship. To learn more about this data relationship,
```

   see the Wikipedia entry about :wikipedia:`One-to-one (data model)
   <w/index.php?title=One-to-one_(data_model)&oldid=1096960092>`.

Aggregation Task Summary
~~~~~~~~~~~~~~~~~~~~~~~~

This aggregation tutorial demonstrates how to join data from two
collections related to product information and customer orders.
The aggregated result is a list of all orders placed in 2020 that
includes the product details associated with each order.

This example uses two collections:

- ``orders``, which contains documents describing an order
  for a product from a shop
- ``products``, which contains documents that describe the available

>   products from a shop

Reviewer: Nit:
```suggestion
  products in a shop
```

An order can only contain one product, so the aggregation uses a
one-to-one join to match an order document to the document for the ordered
product. The collections are joined by a field called ``product_id``
that exists in documents in both collections.

Before You Get Started
----------------------

> Before you start following this tutorial, set up the aggregation
> tutorial template app in file called ``agg_tutorial.js`` by following the
> instructions in the :ref:`node-agg-tutorial-template-app` section of the
> Aggregation Tutorials landing page.

Reviewer: ```suggestion
Before you begin this tutorial, follow the instructions in the 
:ref:`node-agg-tutorial-template-app` section to set up the aggregation 
tutorial template app.
```

> Get a handle on the ``products`` and ``orders`` collections:

Reviewer: Maybe you're using standard phrasing I'm unfamiliar with, but I think this reads clearer:

```suggestion
Create handles for the ``products`` and ``orders`` collections:
```

Author: changed wording
